### ISSUE TITLE
Throws on floats in arrays in objects

### ISSUE THREAD

**Issue by cmeeren** [j-brooke/FracturedJson#33]
Minimal repro:

```f#
FracturedJson.Formatter().Reformat("""{"a":[{"b":1.2}]}""")
```

This throws:

```
System.FormatException
The input string '1.2' was not in a correct format.
   at System.Number.ThrowFormatException[TChar](ReadOnlySpan`1 value)
   at System.Double.Parse(String s)
   at FracturedJson.Formatting.TableTemplate.MeasureRowSegment(JsonItem rowSegment)
   at FracturedJson.Formatting.TableTemplate.MeasureRowSegment(JsonItem rowSegment)
   at FracturedJson.Formatting.TableTemplate.MeasureRowSegment(JsonItem rowSegment)
   at FracturedJson.Formatting.TableTemplate.MeasureTableRoot(JsonItem tableRoot)
   at FracturedJson.Formatter.FormatContainerTable(JsonItem item, Int32 depth, Boolean includeTrailingComma)
   at FracturedJson.Formatter.FormatContainer(JsonItem item, Int32 depth, Boolean includeTrailingComma)
   at FracturedJson.Formatter.FormatItem(JsonItem item, Int32 depth, Boolean includeTrailingComma)
   at FracturedJson.Formatter.FormatTopLevel(IEnumerable`1 docModel, Int32 startingDepth, IBuffer buffer)
   at FracturedJson.Formatter.Serialize[T](T obj, Int32 startingDepth, JsonSerializerOptions serOpts)
```

I get the same exception if I first parse the document to a `JsonDocument` and then serialize this using `FracturedJson.Formatter().Serialize()`.

**Comment by cmeeren** [comment]
In case it's relevant, my current culture is `nb_NO` which uses `,` as decimal separator, but hopefully FracturedJson does not depend on the current culture (if so, it's likely a bug).

**Comment by cmeeren** [comment]
I can confirm that the error is that FacturedJson uses the current culture. If I call

```f#
System.Globalization.CultureInfo.CurrentCulture <- System.Globalization.CultureInfo.InvariantCulture
```

first, then it works.

FracturedJson must be updated to not depend on the current culture.

**Comment by j-brooke** [comment]
This is a big embarrassing bug.  Thanks for reporting it (and for the helpful analysis).

**Comment by j-brooke** [comment]
Fixed in version 4.0.2 of the .NET library.  I rebuilt the web formatter too, just in case.  As far as I can tell, this bug doesn't apply to the JS or Visual Studio Code versions, but if anyone finds differently, please let me know.
